client/js/components/map/publicdetail/Map.vue

```diff
@@ -943,38 +943,37 @@ export default {
                   url: this.getFeatureInfoUrlPlanningArea
                 })
                   .then(responsePr => {
-                    // If we can't check the procedure we want to get the featureInfos anyway
-                    if (responsePr.data.code === 100 && responsePr.data.success && responsePr.data.body !== null && responsePr.data.body !== '') {
-                      /* Check if coordinates are in the area of the current procedure, but only if planningarea is not set to 'all' */
-                      if (responsePr.data.body.id !== this.procedureId && this.procedureSettings.planningArea !== 'all' && hasPermission('feature_map_new_statement')) {
-                        /*
-                         * Roll back to this one when we can handle procedure versions
-                         * let popUpContent = Translator.trans('procedure.move.to.participate', {name: responsePr.body.name}) +
-                         *     '<a class="btn btn--primary float-right u-mt-0_5 u-mb-0" href="' + Routing.generate('DemosPlan_procedure_public_detail', {'procedure': responsePr.body.id}) + '">' +
-                         *     Translator.trans('procedure.goto') +
-                         *     '</a>';
-                         */
-                        const popUpContent = Translator.trans('procedure.move.to.list') +
-                        '<a class="' + this.prefixClass('btn btn--primary float-right u-mt-0_5') + '" href="' + Routing.generate('core_home') + '">' +
-                        Translator.trans('procedures.all.show') +
-                        '</a>'
-                        this.showPopup('contentPopup', {
-                          title: Translator.trans('procedure.not.in.scope'),
-                          text: popUpContent
-                        }, coordinate)
-                      } else {
-                        this.getFeatureInfoAndShowPriorityAreaPopup(remappedUrl, coordinate)
-                      }
+                    /*
+                     * If we can't check the procedure we want to get the featureInfos anyway and
+                     * if coordinates are in the area of the current procedure, but only if planningarea is not set to 'all'
+                     */
+                    if (responsePr.data.code === 100 &&
+                      responsePr.data.success &&
+                      responsePr.data.body?.id !== this.procedureId &&
+                      this.procedureSettings.planningArea !== 'all' &&
```

Comment:
I think you need to remove `this.procedureSettings.planningArea` at all from this if

Reply:
IT depends what we want to archive. IF I do so, that would change the logic and eventually display the wrong popup IF the planningArea is set to "all"

Reply:
hm, ok. I just saw `all` and empty string in the database

Reply:
cant say. especially not for Prod